package.json

```
@@ -62,7 +62,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.8.7",
     "remark-cli": "^11.0.0",
-    "stylelint": "^15.5.0"
+    "stylelint": "stylelint/stylelint"
```

Wait for `15.6.0` release:

```suggestion
    "stylelint": "^15.6.0"
```